administrator-manual/en/webmail.rst

```diff
@@ -28,6 +28,16 @@ For example, given a server with IP address *192.168.1.1* and name *mail.mydomai
                 a dedicated user account in AD is required by the module to be fully
                 operational! See :ref:`join-existing-ad-section`.
 
+.. only:: nscom
+
+   .. note::
+
+       You can find a more recent version of roundcubemail in NethForge, you must remove first ``nethserver-roundcubemail`` and 
+       install ``nethserver-roundcubemail-next`` from NethForge. We migrate the database to rh-mariadb105 SCL and we use PHP 7.3 SCL.
```

Comment:
```suggestion
       install ``nethserver-roundcubemail-next`` from NethForge.
```

I guess a user doesn't care what is under the hood if it just works :)

Reply:
done